scripts/Follower.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python2
```

Comment:
I've learned that this line is only necessary if you're intending on making your file an executable - this way, the command line knows where to go to find which program to execute it with. Since this program is being used from ROS, this line is probably unnecessary. Just some info